bundle/src/main/java/com/adobe/acs/tools/explain_query/impl/ExplainQueryServlet.java

```diff
@@ -252,11 +264,12 @@ private JSONObject explainQuery(final Session session, final String statement,
                 List<String> logs = logCollector.getLogs(collectorKey);
```

Comment:
This Thread's collectorKey used to get the logs for this execution (and not other concurrent executions of Explain Query)